modules/infrastructure-moving-registry.adoc

```diff
@@ -15,6 +15,7 @@ You configure the registry Operator to deploy its pods to different nodes.
 
 . View the `config/instance` object:
 +
+[source,terminal]
```

Comment:
Do we want an .Example output after _The output resembles the following text:_ on line 23?